modules/viewing-a-project-using-the-web-console.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// applications_and_projects/working-with-projects.adoc
+
+[id='viewing-a-project-using-the-web-console-{context}']
+= Viewing a project using the web console
+
+.Procedure
+
+. Navigate to *Home* -> *Projects*.
+
+. Select a project to view.
```

Comment:
Wondering if we should describe the two different views on this page, the Resource List which shows workloads in the project and the Dashboard which shows metrics and details about the project iself.